backbone.js

```diff
@@ -68,8 +68,10 @@
   // form param named `model`.
   Backbone.emulateJSON = false;
 
-  // Proxy Underscore methods to a Backbone class' prototype using a
-  // particular attribute as the data argument
+  // Proxy Backbone class methods to Underscore functions, wrapping the model's
+  // attributes object or collection's models array behind the scenes.
```

Comment:
Yeah good call, I wasn't sure since it's kind of inconsistent at the moment. Same with Model vs model and Collection vs collection.